src/components/profile/sectionrocket/sectionrocket.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import './sectionrocket.css';
+import { useSelector } from 'react-redux';
 
 function SectionRocket() {
+  const rockets = useSelector((state) => state.rocket.filter((ro) => ro.reserved));
   return (
     <>
       <div className="rockets__container">
-        <h3 className="rockets__headline">My Rockets</h3>
+        {rockets.length === 0 ? (<span>No</span>) : (<h3 className="rockets__headline">My Rockets</h3>)}
```

Comment:
- [Optional] You can have your message that shows no rockets as "No Reserved Rocket" which will be more descriptive and good for UI